lib/checkexceptionsafety.cpp

```diff
@@ -324,6 +322,11 @@ void CheckExceptionSafety::noexceptThrowError(const Token * const tok)
     reportError(tok, Severity::error, "throwInNoexceptFunction", "Exception thrown in function declared not to throw exceptions.", CWE398, Certainty::normal);
 }
 
+void CheckExceptionSafety::entryPointThrowError(const Token * const tok)
+{
+    reportError(tok, Severity::error, "throwInEntryPoint", "Exception thrown in function that is an entry point.", CWE398, Certainty::normal);
```

Comment:
I suggest something like:
```
Unhandled exception is thrown in function that is an entry point.
```
We only warn if it's unhandled right?

This is undefined behavior right? To throw an exception that is not catched..